transform/mattermost-analytics/models/intermediate/product/configs/int_mattermost2__config_ldap.sql

```diff
@@ -0,0 +1,22 @@
+{{
+    config({
+        "materialized": "incremental",
+        "unique_key": ['event_id'],
+        "incremental_strategy": "delete+insert",
+        "cluster_by": ['received_at_date'],
+        "snowflake_warehouse": "transform_l"
+    })
+}}
+
+select 
+     {{ dbt_utils.star(ref('stg_mattermost2__ldap')) }}
+     , cast(received_at AS date) AS received_at_date
+     , server_id is not null as has_segment_telemetry_data
+    from {{ ref('stg_mattermost2__ldap') }} tmp
+    where
+    -- Exclude items with missing timestamps
+       timestamp is not null     
```

Comment:
Since this is injected by Rudder/segment if missing, it shouldn't be null. Are there any example cases that this happened? Otherwise we can remove it.